examples_tf2_py/examples_tf2_py/async_waits_for_transform.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+
+import math
+
+from geometry_msgs.msg import TransformStamped
+import rclpy
+from rclpy.duration import Duration
+from rclpy.node import Node
+from rclpy.time import Time
+from tf2_ros import LookupException
```

Comment:
I don't think we use `Duration`, `Time`, or `LookupException` in the code below; please remove the imports.

The same goes for the other files in this PR, where we should only import what we are using.

Reply:
Added linter tests in e0479dc3d7401b8e2607352954d08a5e0e59c150 and fixed linter errors in 48ca46e4c1a7d81d13799d725a0fbdeafbbb7cac